### PR TITLE
Deprecation Guide for Globals Resolver

### PR DESCRIPTION
**Pull request by Gaurav0**
This is my first draft for #153 

[Rendered](https://github.com/Gaurav0/deprecation-app/blob/deprecation_guide_globals_resolver/content/ember/v3/deprecate-globals-resolver.md)

### CODE REVIEW

File: content/ember/v3/deprecate-globals-resolver.md [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+title: Use ember-cli resolver rather than legacy globals resolver
+until: '4.0.0'
+since: '3.2'
+---
+
+Over the past years we have transitioned to using Ember-CLI as the main way to compile Ember apps. The globals resolver is a holdover and primarily facilitates use of Ember without Ember-CLI.
+
+If at all possible, it is highly recommended that you transition to using ember-cli to build your Ember applications. Most of the community already uses it and it provides many benefits including a rich addon ecosystem.
+
+However, if you dp have a custom build system, or are using Ember App Kit,
```
Comment (knownasilya) [inline content/ember/v3/deprecate-globals-resolver.md]: dp -> do

**Comment by jenweber** [comment]
Thanks for writing this up! 1 or 2 edits to make:
- please replace Slack with a link to Discord, now that's all finalized
- take a look at this screenshot below. I think the app's formatting is possibly not what you might have expected compared to normal markup. I have no idea why bold does nothing in this app. You might want to choose a different way to emphasize the text.

**Comment by Gaurav0** [comment]
@jenweber I cannot see the screenshot.

**Comment by Gaurav0** [comment]
rebased

**Comment by jenweber** [comment]
Awesome! Thank you for also fixing the bolded styling issue.

**Pull request merged**